appliance_hardening_guide/_topics/db_ssl.md

```diff
@@ -106,6 +115,9 @@ The database appliance now accepts only connections from connecting appliances t
         vmdb_production=#
 
     The `psql` displays information about the SSL connection, which indicates that the configuration succeeded. Enter `\q` to leave `psql`.
+    To understand the values for `PGSSLMODE` please see [postgress sslmode documentation].
+
+[postgres ssl documentation](https://www.postgresql.org/docs/9.0/libpq-connect.html#LIBPQ-CONNECT-SSLMODE)
```

Comment:
Should this use the v12 link? https://www.postgresql.org/docs/12/libpq-connect.html#LIBPQ-CONNECT-SSLMODE